vpr/src/route/route_timing.cpp

```diff
@@ -786,7 +804,7 @@ bool try_timing_driven_route_tmpl(const t_router_opts& router_opts,
             VTR_LOG("Critical path: %g ns\n", 1e9 * best_routing_metrics.critical_path.delay());
         }
 
-        VTR_LOG("Successfully routed after %d routing iterations.\n", itry);
+        VTR_LOG("timing Successfully routed after %d routing iterations.\n", itry);
```

Comment:
Not sure why the message is changed here.

Reply:
This also was left over from when I began trying to understand VTR. It will be removed.